cvat-ui/src/utils/mousetrap-react.tsx

```diff
@@ -15,6 +15,7 @@ export interface KeyMapItem {
     scope: ShortcutScope;
     nonActive?: boolean;
     applicable?: string[];
+    weight?: number;
```

Comment:
Also, if  it is only for display, I would name it like `displayWeight`. 
Because now the field feels like something related to priority.